optprompt/optprompt.py

```diff
@@ -9,13 +11,10 @@ class OptionPrompterError(Exception):
 class OptionPrompter():
 
     def __init__(self, *args, config_files=None, section_header='defaults', **kwargs):
-        self.opts = list()
-        self.config_files = list()
-        self.config_opts = list()
+        self.opts = []
+        self.config_files = config_files or []
```

Comment:
Won't this set self.config_files to True or False?
I think it needs to be:
self.config_files = config_files if config_files else []

Reply:
Nope, it will return one or the other.
```python
In [1]: foo = 'foo'
In [2]: foo or 'bar'
Out[2]: 'foo'
```